participants.tsv

```diff
@@ -0,0 +1,30 @@
+participant_id	sex	age
+.git	n/a	n/a
+10062	n/a	n/a
+10063	n/a	n/a
+10328	n/a	n/a
+9418	n/a	n/a
+9584	n/a	n/a
+9604	n/a	n/a
+9605	n/a	n/a
+9611	n/a	n/a
+9612	n/a	n/a
+9613	n/a	n/a
+9669	n/a	n/a
+9709	n/a	n/a
+9710	n/a	n/a
+LICENSE	n/a	n/a
+README.md	n/a	n/a
+sub-10062	n/a	n/a
```

Comment:
you can get these information from: https://docs.google.com/document/d/1jDFZrpJ8nWunP9HjUTN5yDwT4cbnM5zaLDgDliLyMxc/edit#